docs/project/software-required.md

```diff
@@ -65,16 +65,15 @@ To check if `docker` is already installed on Linux:
     $ docker --version
     Docker version 1.5.0, build a8a31ef
 
-On Mac OS X or Windows, you should have installed Boot2Docker which includes
-Docker. You'll need to verify both Boot2Docker and then Docker. This
+On Mac OS X or Windows, you should have installed Docker Toolbox which includes
+Docker. You'll need to verify both Docker Machine and then Docker. This
```

Comment:
s/and then/and/ - (not introduced by this PR) "and then" suggests order is important, which it isn't (?)